internal/lefthook/runner/runner.go

```diff
@@ -351,7 +351,9 @@ func (r *Runner) runCommands() {
 func (r *Runner) runCommand(name string, command *config.Command) {
 	args, err := r.prepareCommand(name, command)
 	if err != nil {
-		logSkip(name, err.Error())
+		if !(r.SkipSettings.SkipSkip() && err.Error() == "no files for inspection" || err.Error() == "no matching staged_files") {
+			logSkip(name, err.Error())
+		}
```

Comment:
I think it's better put this code in `logSkip` function and make it a `Runner`'s method in order to be able to check SkipSettings. There are plenty places that log about skipping :)